vermouth/tests/test_dssp.py

```diff
@@ -30,6 +30,272 @@
                       'SCSSEEEEEESTTHHHHTTTSSHHHHHHHHHHHHHTTCCEEEEEC')
 
 
+# TODO: The code is very repetitive. There may be a way to refactor it with
+# clever use of parametrize and fixtures.
```

Comment:
I think you can `@parametrize` your `@fixtures`. And it should magically work.
https://docs.pytest.org/en/latest/fixture.html#parametrizing-fixtures

Reply:
You can parametrize fixtures in some capacity. But here I would need to think about a clever architectures of the tests that does not obfuscate them.